2_crosswalk_munge/src/merge_id_crosswalks.R

```diff
@@ -0,0 +1,21 @@
+merge_id_crosswalks <- function(
```

Comment:
The name of this function (and the target it returns) is fairly generic, but it is specific to a WQP and NHD crosswalk. Since crosswalks are likely to exist for other identifiers (e.g., MN lake identifiers to and from NHD ids, LAGOS IDs to/from NHD), should this function name be more specific?

Reply:
I was looking at Sam's PR to see how she handled DOWs (the MN lake IDs that will need to be referenced to NHD) but maybe that part isn't implemented.

Reply:
I was thinking of this as just a preliminary implementation of merge_id_crosswalks, such that we'd be able to add DOWs and the WI IDs to this same function once we've made them available.

I chatted briefly with Sam about those other crosswalks before she left for AL until Monday - she referred me to https://github.com/USGS-R/pgml_temperature_prediction/tree/master/lib/crosswalks and David, but I haven't taken on that piece yet. Just getting the NHD and WQP together is what I need to be able to develop 6_temp_wqp_fetch, so that's why I've stopped here for now.

Reply:
Actually, I have a comment related to this here: https://docs.google.com/document/d/1iwwhVUIjcxOFy1GT5i9x_yLAYJbldLFOYuGVpZxS8Vw/edit?disco=AAAACCQpxJM - please do weigh in there when you get a chance

Reply:
Looks/sounds good. I commented in the doc.